### PR TITLE
codeview modify

### PR DESCRIPTION
**Pull request by zhilong-xu**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: delfin/drivers/fujitsu/eternus/eternus_stor.py [hunk 1]
```diff
@@ -395,78 +395,82 @@ def get_access_url():
...
+    def get_fc_sas_initiator(self, host_status, initiator_list, command,
+                             encapsulate_data_total, name_count,
+                             wwn_count, initiator_type):
+        host_fc_list = self.get_data(command)
+        for host_fc in host_fc_list:
+            if len(host_fc) < encapsulate_data_total:
                 continue
-            fc_name = host_fc[consts.HOST_NAME_COUNT]
-            fc_wwn = host_fc[consts.HOST_WWN_COUNT]
+            fc_name = host_fc[name_count]
```
Comment (ghca-cxl) [inline delfin/drivers/fujitsu/eternus/eternus_stor.py:420]: 建议避免使用dict[key]的方式获取值

File: delfin/drivers/fujitsu/eternus/eternus_stor.py [hunk 2]
```diff
@@ -395,78 +395,82 @@ def get_access_url():
...
+
+    def get_fc_sas_host(self, host_list, host_status, command,
+                        encapsulate_data_total, name_count, type_count):
+        host_fc_list = self.get_data(command)
+        for host_fc in host_fc_list:
+            if len(host_fc) < encapsulate_data_total:
                 continue
-            fc_name = host_fc[consts.HOST_NAME_COUNT]
-            os = host_fc[consts.HOST_TYPE_COUNT].lower()
+            fc_name = host_fc[name_count]
```
Comment (ghca-cxl) [inline delfin/drivers/fujitsu/eternus/eternus_stor.py:472]: 建议避免使用dict[key]的方式获取值

File: delfin/drivers/fujitsu/eternus/eternus_stor.py (hunk 1 above)
Comment (zhilong-xu) [reply delfin/drivers/fujitsu/eternus/eternus_stor.py:420]: host_fc is a list

File: delfin/drivers/fujitsu/eternus/eternus_stor.py (hunk 2 above)
Comment (zhilong-xu) [reply delfin/drivers/fujitsu/eternus/eternus_stor.py:472]: host_fc is a list

**Pull request merged**